collect_app/src/main/java/org/odk/collect/android/utilities/TimerLogger.java

```diff
@@ -27,6 +27,14 @@
  */
 public class TimerLogger {
 
+    private static AsyncTask saveTask = null;
```

Comment:
I don't really like this change, either. It feels easier to reason about this when the fields are closer to the constructor and methods.

Reply:
@lognaturel Android Studio also suggests (using IDE's auto format feature) to put all `variables` at the top, then the `constructor` and `methods`, and after that `enums` and `classes`. But since the scope of this PR is just to fix the variables' position, I thought moving the methods and constructor might make it harder to review the PR. But I am too in the favor of moving them up. So if you agree, I will make those changes as well in the  same PR.